runtime/gc_structs/ClassLoaderClassesIterator.hpp

```diff
@@ -62,6 +62,16 @@ class GC_ClassLoaderClassesIterator
 		ANONYMOUS_CLASSES
 	};
 	ScanModes _mode; /**< indicate type of classes to be iterated */
+	J9Class *_iterateArrayClazz;
+	J9Class *_startingClass;
+	enum ArrayClassState {
+		state_valueTypeArray = 0, /* setup value type array list walk */
+		state_valueTypeArrayList, /* value type array list walk */
+		state_array, /* setup array list walk */
+		state_arrayList, /* array list walk */
+		state_done
+	};
+	ArrayClassState _arrayState;
```

Comment:
Would you please add short annotation to added variables `_iterateArrayClazz`, `_startingClass` and `_arrayState` in format `/**< ... */`?

Reply:
Updated.